wn/_add.py

```diff
@@ -73,6 +73,7 @@
 def add(
     source: AnyPath,
     progress_handler: Optional[Type[ProgressHandler]] = ProgressBar,
+    lmf_progress_handler: Optional[Type[ProgressHandler]] = ProgressBar,
```

Comment:
I still don't like the idea of having a separate progress handler class. I think it is unlikely that a user will want a separate class (as opposed to separate instances) for each individual process. Let's just reuse `progress_handler` to instantiate the `progress_lmf` below.

Reply:
You are totally right.